test/postcode_checker/index.js

```diff
@@ -0,0 +1,60 @@
+// this is to simulate the upload evidence api. It's not part of the main app.
```

Comment:
This is a cool thing for us to have to mock the postcode api! Maybe you could make a new folder called `mock` and move it into there?